src/components/Header/index.tsx

```diff
@@ -42,6 +42,9 @@ const Header = ({ t }: { t: TFunction }) => {
         <CustomNavLinkSmall onClick={() => scrollTo("product")}>

```

Comment:
- **Runtime error check**  
  There is no error handling for the `scrollTo` function. If the function fails, it may cause a runtime error.  
- **Optimization**  
  The `scrollTo` function is called multiple times. It would be more efficient to create a function that takes the id as a parameter and calls `scrollTo` with that id.  
- **Security issue**  
  No security issues found.